frontend/packages/kubevirt-plugin/src/components/vms/menu-actions.tsx

```diff
@@ -189,7 +188,6 @@ const menuActionCdEdit = (
 
 export const menuActionDelete = (kindObj: K8sKind, vm: VMKind): KebabOption => ({
   label: `Delete ${kindObj.label}`,
-  href: getVMLikeModelListPath(false, getNamespace(vm)),
```

Comment:
I don't see a reason why this shouldn't be here. I know we are redirecting to virtualization by default now, but with having this we are not polluting the history with redirect

Reply:
a - when we define `href` it hides the filter:
https://bugzilla.redhat.com/show_bug.cgi?id=1828829
b - default delete behaviour does not define href.

> with having this we are not polluting the history with redirect

This is the regular behaviour when deleting objects, we should be as close as possible to how console does things, for example:
https://github.com/openshift/console/blob/master/frontend/public/components/utils/kebab.tsx#L247
https://github.com/openshift/console/blob/master/frontend/packages/console-app/src/components/nodes/menu-actions.tsx#L54

Reply:
ok, then let's remove it if it hides the filter

I just mean the redirect is not done by default in other places